src/test/java/net/sf/jabref/logic/exporter/MSBibExportFormatTestFiles.java

```diff
@@ -37,31 +37,42 @@
 @RunWith(Parameterized.class)
 public class MSBibExportFormatTestFiles {
 
+    public static final String SRC_TEST_RESOURCES = "src/test/resources/";
+
+    private Path importFile ;
     public BibDatabaseContext databaseContext;
     public Charset charset;
     public File tempFile;
     public MSBibExportFormat msBibExportFormat;
-    public BibtexImporter testImporter;
 
+    public BibtexImporter testImporter;
     @Parameter
     public String filename;
-    public Path resourceDir;
 
+    public Path resourceDir;
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
 
 
     @Parameters(name = "{0}")
     public static Collection<String> fileNames() throws IOException, URISyntaxException {
-        try (Stream<Path> stream = Files.list(Paths.get(MSBibExportFormatTestFiles.class.getResource("").toURI()))) {
+        String packageName = MSBibExportFormatTestFiles.class.getPackage().getName();
+        Path path = convertPackageNameToPath(packageName);
+
+        try (Stream<Path> stream = Files.list(path)) {
             return stream.map(n -> n.getFileName().toString()).filter(n -> n.endsWith(".bib"))
                     .filter(n -> n.startsWith("MsBib")).collect(Collectors.toList());
         }
     }
 
+    private static Path convertPackageNameToPath(String packageName) {
+        return Paths.get(SRC_TEST_RESOURCES +packageName.replace('.','/')+"/");
+    }
+
     @Before
     public void setUp() throws Exception {
-        resourceDir = Paths.get(MSBibExportFormatTestFiles.class.getResource("").toURI());
+        resourceDir = convertPackageNameToPath( BibTeXMLExporterTestFiles.class.getPackage().getName());
```

Comment:
Should be `MSBibExportFormatTestFiles.class.getPackage().getName()`, Copy paste mistake?